src/logion/controllers/locrequest.controller.ts

```diff
@@ -114,20 +121,27 @@ export class LocRequestController extends ApiController {
             userIdentity: this.fromUserView(createLocRequestView.userIdentity)
         }
         let request: LocRequestAggregateRoot;
+        let notifyLO: boolean
         if (authenticatedUser.isNodeOwner()) {
             request = await this.locRequestFactory.newOpenLoc({
                 id: uuid(),
                 description,
             });
+            notifyLO = false
         } else {
             request = await this.locRequestFactory.newLocRequest({
                 id: uuid(),
                 description
             });
+            notifyLO = true
         }
         await this.checkIdentityLoc(request.requesterIdentityLocId)
         await this.locRequestRepository.save(request);
         const userIdentity = await this.findUserIdentity(request);
+        if (notifyLO && userIdentity) {
+            this.getNotificationInfo(request.getDescription(), userIdentity)
+                .then(info => this.notificationService.notify(info.legalOfficerEMail, "loc-requested", info.data))
```

Comment:
What's the reason behind using the Promise-like syntax instead of await?

Reply:
Performance. Both LO info fetching and mail sending can/should be done asynchronously. A potential failure of one of those 2 operations will anyway not affect returned response.